plugins/quay/src/components/QuayTagPage/component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useRepository, useTagDetails } from '../../hooks';
+import { useParams } from 'react-router-dom';
+import { QuayTagDetails } from '../QuayTagDetails';
+import { ErrorPanel } from '@backstage/core-components';
+import { useRouteRef } from '@backstage/core-plugin-api';
+import { rootRouteRef } from '../../routes';
+
+type QuayTagPageProps = Record<never, string>;
+
+export const QuayTagPage: React.FC<QuayTagPageProps> = () => {
+  const rootLink = useRouteRef(rootRouteRef);
+  const { repository, organization } = useRepository();
+  const { digest } = useParams();
+  if (!digest) {
+    throw new Error('digest is not defined');
+  }
+  const { loading, value } = useTagDetails(organization, repository, digest);
+  if (loading) {
+    return <div>loading...</div>;
```

Comment:
Can you display `<Progress />` here instead, please? https://backstage.io/storybook/?path=/story/feedback-progress--progress